### PR TITLE
Fixes views without SRID and not populated

### PR DESCRIPTION
**Pull request by lbartoletti**
Using a cast for vw_pipe_schema
Using a cast for export_pipe with a workaround in python

### CODE REVIEW

**Comment by sylvainbeo** [comment]
Looks ok for me. Seeing the 21781 in the sql file, it reminds me that we could think about a way to externalize it.

**Comment by lbartoletti** [comment]
Yes. I didn't understand at first why it was harcoded.

**Comment by haubourg** [comment]
this is #199 
And yes it is important to externalise it

File: update/delta/delta_1.3.2_007_fixes_views.sql [hunk 1]
```diff
@@ -0,0 +1,256 @@
+-- View: qwat_od.vw_export_pipe
+
+-- DROP VIEW qwat_od.vw_export_pipe;
+
+CREATE OR REPLACE VIEW qwat_od.vw_export_pipe AS
```
Comment (elemoine) [inline update/delta/delta_1.3.2_007_fixes_views.sql]: I need to check but I am not sure we need deltas for views.

**Comment by haubourg** [comment]
Correct. no Delta needed for views if they are created via ordinary_data/views scripts

**Comment by lbartoletti** [comment]
Removed. Thanks.

**Comment by haubourg** [comment]
Thanks Loïc, merging!

**Pull request merged**